relay-server/src/actors/processor.rs

```diff
@@ -2283,6 +2284,8 @@ impl EnvelopeProcessorService {
             .project_state
             .has_feature(Feature::SpanMetricsExtractionAllModules);
 
+        let op_denylist = HashSet::from(["clickhouse", "mongodb", "redis", "activerecord"]);
```

Comment:
This constructs a data structure for static data in every run. Instead of this, you could make this a function that uses `match` to check for the four allowed values.

Reply:
I ended up refactoring the conditions so it's just code. Let me know if that's better.